ui/app/helpers/utils/transactions.util.js

```diff
@@ -184,8 +180,8 @@ export function getStatusKey(transaction) {
   }
 
   if (
-    status === TRANSACTION_STATUS_CONFIRMED &&
-    type === TRANSACTION_TYPE_CANCEL
+    status === TRANSACTION_STATUSES.CONFIRMED &&
+    type === TRANSACTION_TYPES.CANCEL
```

Comment:
The hard-coded `failed` and `cancelled` values returned here should be using the constants as well

Reply:
We should use these constants in `tx-state-manager` as well :thinking: That's where the `status` returned below is set.